applications/FluidDynamicsApplication/custom_utilities/fluid_auxiliary_utilities.cpp

```diff
@@ -29,6 +30,35 @@
 namespace Kratos
 {
 
+void FluidAuxiliaryUtilities::CreateSlipMultiPointConstraints(
+    ModelPart& rModelPart,
+    const Variable<array_1d<double,3>>& rUnknownVar,
+    const Flags& rSlipFlag)
+{
+    // Set the normal wall velocity calculation function
+    auto wall_normal_v_func = [&](const NodeType& rNode, const array_1d<double,3>& rUnitNormal){
+        return 0.0;
+    };
+
+    // Call internal function to create the slip MPCs
+    InternalCreateSlipMultiPointConstraints(rModelPart, rUnknownVar, rSlipFlag, wall_normal_v_func);
+}
+
+void FluidAuxiliaryUtilities::CreateSlipMultiPointConstraints(
+    ModelPart& rModelPart,
+    const Variable<array_1d<double,3>>& rUnknownVar,
+    const Variable<array_1d<double,3>>& rWallVelocityVar,
+    const Flags& rSlipFlag)
+{
+    // Set the normal wall velocity calculation function
+    auto wall_normal_v_func = [&](const NodeType& rNode, const array_1d<double,3>& rUnitNormal){
```

Comment:
mmm, i would capture nothing. You have no need for it, so it is way better if you don't use it

Reply:
True. Thanks for spotting.

Reply:
Done.